run/types/testing.ts

```diff
@@ -247,7 +250,7 @@ export type AccessibilityId =
   | 'Photo, May 01, 1999, 7:00 AM'
   | 'profile_picture.jpg, 27.75 kB, May 2, 1999'
   | 'profile_picture.jpg, 27.75 kB, May 1, 1999'
-  | 'Photo taken on May 2, 1999, 7:00:00 AM'
+  | `Photo taken on May 2, 1999 7:00:00 AM`
```

Comment:
why the backtick? prettier should fix this for you already

Reply:
i'm not sure, but i removed the backticks